activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -1025,6 +1032,10 @@ def reverse_sql_order(order_query)
         end
       end
     end
+    
+    def order_uses_function_with_multiple_arguments?(order)
```

Comment:
function is more correct. Maybe rename to `uses_sql_function_with_multiple_arguments?` to make it more clear?